gravitee-am-repository/gravitee-am-repository-jdbc/src/main/resources/liquibase/changelogs/v4_4_0/4.4.0-add-account-access-tokens-table.yml

```diff
@@ -0,0 +1,34 @@
+databaseChangeLog:
+  - changeSet:
+      id: 4.4.0-add-account-access-tokens-table
+      author: GraviteeSource Team
+      validCheckSum: ANY
+      preConditions:
+        - onFail: MARK_RAN
+        - not:
+            - tableExists:
+                tableName: account_access_tokens
+      changes:
+        - createTable:
+            tableName: account_access_tokens
+            columns:
+              - column: { name: id, type: nvarchar(64), constraints: { nullable: false } }
+              - column: { name: reference_type, type: nvarchar(64), constraints: { nullable: false } }
+              - column: { name: reference_id, type: nvarchar(255), constraints: { nullable: false } }
+              - column: { name: user_id, type: nvarchar(64), constraints: { nullable: false } }
+              - column: { name: name, type: nvarchar(255), constraints: { nullable: false } }
+              - column: { name: token, type: nvarchar(64), constraints: { nullable: false } }
+              - column: { name: created_at, type: timestamp(6), constraints: { nullable: false } }
+              - column: { name: updated_at, type: timestamp(6), constraints: { nullable: false } }
+        - addPrimaryKey:
+            constraintName: pk_account_access_tokens
+            columnNames: id
+            tableName: account_access_tokens
+
+        - createIndex:
+            columns:
+              - column:
+                  name: user_id
+            indexName: idx_account_access_tokens_user_id
+            tableName: account_access_tokens
+            unique: false
```

Comment:
**todo**: IMO we should introduce an index based on the token because we can't assume how many token will be created for an organization. As the token will not bring the userId, we will have to load all the otken for a given organization without this index